lib/tic_tac_toe.rb

```diff
@@ -0,0 +1,26 @@
+module TicTacToe
+end
+
+require_relative "./tic_tac_toe/core_extensions.rb"
+require_relative "./tic_tac_toe/board.rb"
+require_relative "./tic_tac_toe/cell.rb"
+require_relative "./tic_tac_toe/game.rb"
+require_relative "./tic_tac_toe/player.rb"
```

Comment:
Up to that line it's ok. I think the rest of the file should go into the executable in *bin/tic_tac_toe..*

Reply:
Updated